extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveServerConfig.java

```diff
@@ -7,8 +7,11 @@
 public class ResteasyReactiveServerConfig {
 
     /**
-     * Set this to override the default path for JAX-RS resources if there are no
-     * annotated application classes.
+     * Set this to define the application path that serves as the base URI for all
+     * JAX-RS resource URIs provided by Path annotations when there are no
+     * {@code @ApplicationPath} annotation defined on {@code Application} classes.
+     * <p>
+     * This value is always resolved relative to {@code quarkus.http.root-path}.
      */
     @ConfigItem(defaultValue = "/")
```

Comment:
Is it confusing that we're saying it's a relative path and then the default is an absolute?

Reply:
This is a substitute for `@ApplicationPath` .. it is JAX-RS. It will ALWAYS be interpreted relative to http.root-path. Always

Reply:
I'm trying to make that unambiguous .. by reference more JAX-RS things in JAX-RS-y language.

Reply:
@stuartwdouglas .. we could remove the default or leave it empty or.. ?